ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -658,5 +659,131 @@ public void TestOmittedParameters()
                 Assert.AreEqual(2, value);
             }
         }
+
+        [Test]
```

Comment:
Sorry, just realised that these tests should actually go in the `TextTests` class, because the function itself appears in the `CalcEngine.Text` class. 

Also look at e.g. the `Text_DateFormats` test and rather use `TestCase` attributes to test the different cases instead of multiple asserts.